tests/utils/stacks/tests.py

```diff
@@ -95,11 +95,40 @@ def test_frame_allowance(self):
         assert results['frames'][8]['filename'] == '8'
         assert results['frames'][9]['filename'] == '9'
 
+class FailLoader():
+    '''
+    Recreating the built-in loaders from a fake stack trace was brittle.
+    This method ensures its testing the path where the loader is defined
+    but fails with known exceptions.
+    '''
+    def get_source(self, module_name):
+        if '.py' in module_name:
```

Comment:
Isn't this going to cause a severe issue if you just had a module named `foo.pythings.whatever`?